docs/supplychain-sbom.md

```diff
@@ -411,9 +416,13 @@ When doing business within the European Economic Area (EEA), has the duty to ens
 
 * See also [Owner](#owner)
 
-| Do | Field name                    | Required | Data type | CycloneDX (PRE-PROPOSAL; UNSUPPORTED)                          | SPDX | Required by                        |
+<!--
+SPDX 2.3 doesn't support the CE fields. SPDX 3.0 should be used at a future date
```

Comment:
Yeah, I'm not expecting any of the standards to support the CE fields, but it's pretty certain they'll be required soon, so they are here as a reminder. :grin: